CONTRIBUTING.md

```diff
@@ -54,7 +54,7 @@ documentation.
 1. [Fork](https://help.github.com/articles/fork-a-repo) SunPy on GitHub
 1. Install the developer version of SunPy using git. 
 1. Make your changes and push to your fork.
-1. Create a .
+1. Create a pull request.
```

Comment:
might be worth linking to: https://help.github.com/articles/creating-a-pull-request-from-a-fork/